cloud/disk_manager/test/acceptance/test_runner/eternal_acceptance_test_runner.py

```diff
@@ -1,29 +1,53 @@
+import argparse
 import contextlib
 import logging
 import math
-
+import re
 from concurrent.futures import ThreadPoolExecutor
+from datetime import timedelta
+
+from cloud.blockstore.pylibs import common
+from cloud.blockstore.pylibs.ycp import YcpWrapper
 from .base_acceptance_test_runner import BaseAcceptanceTestRunner, \
     BaseTestBinaryExecutor
-
+from .cleanup import BaseResourceCleaner
 from .lib import (
     check_ssh_connection,
     size_prettifier,
     Error,
 )
 
-from cloud.blockstore.pylibs import common
-
-
 _logger = logging.getLogger(__file__)
 
 
 class EternalAcceptanceTestBinaryExecutor(BaseTestBinaryExecutor):
     _entity_suffix = 'eternal'
 
 
+class EternalTestCleaner(BaseResourceCleaner):
+    def __init__(self, ycp: YcpWrapper, args: argparse.Namespace):
+        super(EternalTestCleaner, self).__init__(ycp, args)
+        test_type = args.test_type
+        disk_size = size_prettifier(args.disk_size * (1024 ** 3)).lower()
+        disk_blocksize = size_prettifier(args.disk_blocksize).lower()
+        disk_name_pattern = re.compile(
+            fr'^acceptance-test-{test_type}-'
+            fr'{disk_size}-'
+            fr'{disk_blocksize}-[0-9]+$',
+        )
+        self._entity_ttls = self._entity_ttls | {
+            'disk': timedelta(days=5),
```

Comment:
Lets use same ttl for other resources?

Reply:
In this case we delete VM and disks only. We inherit a pattern for the instance, and ttl for the instance. Instances are to be removed after the test anyway AFAIR.

Reply:
Lets have separate hard-coded TTL for instances, other resources might have single value as TTL